creating-pull-requests.md

```diff
@@ -0,0 +1,131 @@
+# Creating a Pull Request
+
+This page goes over the steps for creating a Pull Request (PR), including some dos and don’ts.
+
+> Note: the terms _Pull Request_ and _Merge Request_ can be used interchangeably. The same is true for the terms _issue_, _ticket_, and _stories_.
+
+## Before you start working
+
+Make sure that there is an issue created and assigned to you. Work on a feature branch that is named accordingly. It might also be a good idea to validate the approach you intend to take in case it is a big or complex issue.
+
+## When to create a Pull Request?
+
+The two main reasons to create a pull request are when your work is complete, or when you want early feedback on your work.
+
+### When your work is complete
+
+You finished the work on an issue in a dedicated feature branch, and you’re ready to have your code merged into `develop`. Before doing that, others will review it according to the Frontend Pull Request Reviewer Checklist. This process is also useful for others to learn about the changes you have made. The pull request can be merged after the work is reviewed and all of the feedback has been addressed.
```

Comment:
Once the branch is merged, do we usually delete it remotely?

Reply:
added mention to delete the branch after merge!